src/main/resources/marytts/language/hsb/formatRules.txt

```diff
@@ -174,22 +174,6 @@ x.x: << koma >>;
 80: [>>a]wosomdźesate;
 90: [>>a]dźewjećdźesate;
 100: stote;
-%%alt-hundreds:
-1100: jědnaćestow;
-1101-1199: jědnaćestow[ >>];
-1200: dwanaćestow;
-1201-1299: dwanaćestow[ >>];
-1300: třinaćestow;
-1301-1399: třinaćestow[ >>];
-1400: štyrnaćestow;
-1401-1499: štyrnaćestow[ >>];
-1500: pjatnaćestow;
-1501-1599: pjatnaćestow[ >>];
-1600: šěsnaćestow;
-1601-1699: šěsnaćestow[ >>];
-1700: sydomnaćestow;
-1701-1799: sydomnaćestow[ >>];
-1800: wosomnaćestow;
 1801-1899: wosomnaćestow[ >>];
 1900: dźewjatnaćestow;
 1901-1999: dźewjatnaćestow[ >>];
```

Comment:
These three lines at the end still cause the error
```
java.lang.IllegalArgumentException: Illegal character - in rule descriptor
```
for me, but if I remove them, the tests pass.

Reply:
True, I just deleted them. Thanks for checking!